### DISCUSSION TITLE
calls to analogX and digitalX

### DISCUSSION THREAD

**Discussion by reymalahay** [bxparks/UnixHostDuino#87] [Q&A]
hi there,

i have combed through the various issues and discussions in [AUnit](https://github.com/bxparks/AUnit), especially [this discussion](https://github.com/bxparks/AUnit/discussions/83)

i appreciate that there is a way to [mock digitalWrite and digitalRead calls](https://github.com/bxparks/EpoxyDuino#MockDigitalReadDigitalWrite)

is it possible to have fine-grained control of what `digitalReadValue` can return?

also, how does one make mock `analogX` invocations, not just `analogRead`?

please advise.

thanks in advance,
rey malahay

**Comment by bxparks** [comment]
As explained in another thread, you are flooding me with Issues and Discussions: 14 tickets in 3 weeks. Even just reading them takes time and energy, to figure out whether it is a valid Issue, or whether it belongs in a Discussion. Some of your posts have been duplicates. Some have been filed in the wrong project. I just don't have the time and energy for your tickets. So I am going to close all of them, because I won't be able to get to them.
